server/chat-plugins/room-events.ts

```diff
@@ -18,11 +18,28 @@ function formatEvent(event: {eventName: string, date: string, desc: string, star
 	}
 	let ret = `<tr title="${explanation}">`;
 	ret += Chat.html`<td>${event.eventName}</td>`;
+	ret += Chat.html`<td>${(event.aliases ? event.aliases.join(", ") : "")}</td>`;
```

Comment:
```suggestion
	ret += Chat.html`<td>${event.aliases ? event.aliases.join(", ") : ""}</td>`;
```

Missed this; the parentheses are unnecessary with the brackets.